modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerPropertiesTest.java

```diff
@@ -197,4 +199,45 @@ public void testPropertyDefaultQueryTimeout() {
             )
         );
     }
+
+    /**
+     * Check the set command for property 'wal.rebalance.threshold'.
+     */
+    @Test
+    public void testPropertyWalRebalanceThreshold() {
+        assertDistributedPropertyEquals(WAL_REBALANCE_THRESHOLD_DMS_KEY, DFLT_PDS_WAL_REBALANCE_THRESHOLD, true);
+
+        int newVal = 1000;
+
+        assertEquals(
+            EXIT_CODE_OK,
+            execute(
+                "--property", "set",
+                "--name", "wal.rebalance.threshold",
+                "--val", Integer.toString(newVal)
+            )
+        );
+
+        assertDistributedPropertyEquals(WAL_REBALANCE_THRESHOLD_DMS_KEY, newVal, true);
+    }
+
+    /**
+     * Validates that distributed property has specified value across all nodes.
+     * @param propName Distributed property name.
+     * @param expected Expected property value.
+     * @param <T> Property type.
+     */
```

Comment:
Please add empty line before paramerters description and a description for `onlyServerMode`.

Reply:
Will do